models/jules/R/write.config.JULES.R

```diff
@@ -579,12 +587,51 @@ write.config.JULES <- function(defaults, trait.values, settings, run.id) {
 
     ## write namelist
     writeLines(ic.text, con = ic.file)
-
+    
+    if (is.null(settings$run$inputs$poolinitcond$path)){
     ## also load and parse IC dat file
-    ic.dat <- file.path(local.rundir, "initial_conditions.dat")
-    ic.text <- readLines(con = ic.dat, n = -1)
-    ic.text[2] <- paste(ic.text[2]," 5.0 5.0 0.5 0.5 0.5 0.2 0.2 0.2 0.2 0.2 0.0 0.0 0.0 0.0")
-    writeLines(ic.text, con = ic.dat)
+    ## also load and parse IC dat file
+      ic.dat <- file.path(local.rundir, "initial_conditions.dat")
+      ic.text <- readLines(con = ic.dat, n = -1)
+      ic.text[2] <- paste(ic.text[2]," 5.0 5.0 0.5 0.5 0.5 0.2 0.2 0.2 0.2 0.2 0.0 0.0 0.0 0.0")
+      writeLines(ic.text, con = ic.dat)
+    
+    } else if (!is.null(settings$run$inputs$poolinitcond$path)){
+      
+      #Read in path to standard initial conditions file
+      IC.path <- settings$run$inputs$poolinitcond$path
+      
+      ## LAI m2/m2
+      if(!is.null(IC.path)){
+        # Read in netcdf file
+        IC.nc <- ncdf4::nc_open(IC.path)
+        #Get LAI value
+        lai <- try(ncdf4::ncvar_get(IC.nc,"LAI"))
+        #Read in default initial conditions dat file
+        ic.dat  <- file.path(local.rundir, "initial_conditions.dat")
+        ic.dat.text <- readLines(con = ic.dat, n = -1)
+        #For now if number of LAI values does not match number of pfts set them set them across
+        if (npfts!= length(lai)) {
+          ic.dat.text <- gsub("@LAI_DAT@", paste0(rep(lai,npfts), collapse = " "), ic.dat.text)
+          writeLines(ic.dat.text, con = ic.dat)
+        }
+        
+        
+      }else{
+        PEcAn.logger::logger.error("Bad initial conditions filepath")
+      }
+      
+    }
+  }else{ 
+  # If not TRIFFID
+  ic.dat <- file.path(local.rundir, "initial_conditions.dat")
+  ic.dat <- file.path(".", "initial_conditions.dat")
+  ic.text <- readLines(con = ic.dat, n = -1)
+  ic.dat.text <- gsub("@sthuf_dat@", paste0("0.749  0.743  0.754  0.759", collapse = " "), ic.dat.text)
+  ic.dat.text <- gsub("@t_soil@", paste0("276.78  277.46  278.99  282.48", collapse = " "), ic.dat.text)
+  ic.dat.text <- gsub("@snow_tile@", paste0("0.0  0.46  0.0  0.0  0.0  0.0  0.0  0.0  0.0", collapse = " "), ic.dat.text)
+  ic.dat.text <- gsub("@lai_dat@", paste0("5.0  4.0  2.0  4.0  1.0", collapse = " "), ic.dat.text)
+  writeLines(ic.text, con = ic.dat)
```

Comment:
Not following this code at all. If you're not using TRIFFID and don't have a pool IC file, then you **don't need to open/modify the initial_conditions.dat file at all**, so it's not clear why you're inserting a bunch of values here.  If you're not using TRIFFID and you **DO** have a pool IC file, you're ignoring that file when you should be reading it and inserting LAI values.